static/app/utils/performance/contexts/metricsEnhancedPerformanceDataContext.tsx

```diff
@@ -53,6 +53,16 @@ export function MEPDataProvider({
 
 export const useMEPDataContext = _useMEPDataContext;
 
+export function getIsMetricsDataFromResults(results: any, field = '') {
+  const isMetricsData =
+    results?.meta?.isMetricsData ??
+    results?.seriesAdditionalInfo?.[field]?.isMetricsData ??
+    results?.histograms?.meta?.isMetricsData ??
+    results?.tableData?.meta?.isMetricsData ??
+    true;
+  return !!isMetricsData;
+}
```

Comment:
Moved for DRY + added `results.meta.isMetricsData` check here for events total return value. Should add typing to this later, but it'll require every metrics-esque data response to be unioned.